pom.xml

```diff
@@ -66,6 +66,7 @@
               <exclude>deploy/preflight.sh</exclude>
               <exclude>deploy/RPM/CMakeLists.txt</exclude>
               <exclude>deploy/rpm_post.sh</exclude>
+              <exclude>deploy/rpm_post_uninstall.sh</exclude>
```

Comment:
Since this is a new file, we should add the apache license instead of adding it to the exclude list.

Reply:
This was done under the assumption that Apache does not need the license header for simple scripts that don't have considerable IP. In this case, we're only removing some files and a directory, so assuming that it's fine to not explicitly tag the license. Let me know if you feel otherwise.